docs/rancher/csi-driver-support-various-sc.md

```diff
@@ -0,0 +1,84 @@
+---
+sidebar_position: 6
+sidebar_label: Various StorageClass of Harvester CSI Driver 
+title: "Various StorageClass of Harvester CSI Driver"
+keywords:
+  - Harvester
+  - harvester
+  - Rancher Integration
+  - CSI Driver
+  - StorageClass
+---
+
+After the Harvester CSI driver v0.1.15, you can create PVC based on the different StorageClass of Harvester.
+
+## Preparation
+
+It would be best if you did some preparation to support this feature entirely.
```

Comment:
Can you please elaborate on why this pre-configuration is needed here? 
e.g., in order to allow users to balabala, it is recommended to...

Reply:
Updated!